src/DynamoWebServer/Messages/MessageHandler.cs

```diff
@@ -522,7 +523,27 @@ private string GetInOutPortsData(NodeModel node)
                 stringBuilder.Append("],");
             }
 
-            stringBuilder.Append("\"InPorts\": [");
+            if (node is VariableInputNode)
+            {
+                var type = node.GetType();
+                if (type.Name == "PythonNode")
+                {
+                    var script = type.GetProperty("Script").GetValue(node, null).ToString();
+
+                    stringBuilder.Append("\"Script\":\"");
+                    stringBuilder.Append(script.
+                            Replace("\n", "\\n").
+                            Replace("\"", "\\\""));
+                    stringBuilder.Append("\", ");
+                }
+
+                stringBuilder.Append("\"varInputs\": [");
+            }
+            else
+            {
+                stringBuilder.Append("\"InPorts\": [");
+            }
+
```

Comment:
Added DynamoWebServer/Extensions.cs with extensions for CodeBlockNodeModel, VariableInputNode and NodeModel.